ansible/roles/ocp4-workload-pam-fraudmanagement-workshop/tasks/workload_per_project_amqstreamsworkload.yml

```diff
@@ -13,25 +13,51 @@
 
 - name: Get Kafka template
   uri:
-    url: "https://{{ gitea_route.stdout }}/{{ _namespace }}/proactive-fraud-detection-kafka/raw/branch/master/.openshiftio/application.yaml"
+    url: >-
```

Comment:
For these, either increase the max to keep them on one line or use something like this. The way it is currently written is a bit hacky and hard to read.

```
url: >-
  {{ 'https://{0}/{1}/proactive-fraud-detection-kafka/raw/branch/master/.openshiftio/application.yaml'.format(
  gitea_route.stdout, _namespace) }}
```

Same with anything else in this task list that has things broken up like this.

Reply:
Sounds good, will clean up everywhere.